bulk_site_creator/views.py

```diff
@@ -83,17 +83,25 @@ def job_detail(request, job_id):
         'KeyConditionExpression': Key('pk').eq(school_key) & Key('sk').eq(job_id),
         'ScanIndexForward': False,
     }
-    job = table.query(**job_query_params)['Items'][0]
+    job = table.query(**job_query_params)['Items']
+
+    # Update string timestamp to datetime.
+    job[0].update(created_at=parse_datetime(job[0]['created_at']))
+    job[0].update(updated_at=parse_datetime(job[0]['updated_at']))
 
     tasks_query_params = {
         'KeyConditionExpression': Key('pk').eq(job_id),
         'ScanIndexForward': False,
     }
-    tasks = table.query(**tasks_query_params)['Items']
+    tasks = table.query(**tasks_query_params)
+
+    # Add Canvas course URL to tasks with a Canvas course ID and workflow_state != 'fail'.
+    [item.update(canvas_course_url=f"{settings.CANVAS_URL}/courses/{item['canvas_course_id']})")
+     for item in tasks['Items'] if item['canvas_course_id'] and item['workflow_state'] != 'fail']
```

Comment:
I believe something similar is already done in the job_details template. Might want to add the != fail condition there though.